rb/lib/selenium/webdriver/common/manager.rb

```diff
@@ -47,7 +47,7 @@ def add_cookie(opts = {})
 
         opts[:path] ||= '/'
         opts[:secure] ||= false
-
+        opts[:sameSite]
```

Comment:
We don't need this line, do we?

Reply:
Hi @p0deje ,

Thanks for the review. No, it's not required feature works removing this. 
Updating the code